index.js

```diff
@@ -608,14 +601,33 @@ Parser.prototype = {
     };
   },
 
+  parseIncludeFilter: function() {
+    var filter = this.parseFilter(true);
```

Comment:
Inline the bit of `parseFilter` that you need here.  That won't lead to very much code duplication and it removes the "magic" value `true` here.